.env.cluster

```diff
@@ -27,10 +27,10 @@ REPMGR_PARTNER_NODES=santempi-psql-1,santempi-psql-2,santempi-psql-3
 
 # Reverse Proxy - Nginx
 REVERSE_PROXY_INSTANCES=3
+INSECURE=false
 DOMAIN_NAME=domain
-SUBDOMAINS=openhimcomms.domain,openhimcore.domain,openhimconsole.domain,kibana.domain,reports.domain,santewww.domain,santempi.domain,superset.domain,keycloak.domain,grafana.domain
+SUBDOMAINS=openhimcomms.domain,openhimcore.domain,openhimconsole.domain,kibana.domain,reports.domain,santewww.domain,santempi.domain,superset.domain,grafana.domain
```

Comment:
I wonder why did you remove keycloak.domain ?

Reply:
ops sorry merge request didn't notice it